dattri/algorithm/tracin.py

```diff
@@ -19,14 +19,6 @@
 from .base import BaseAttributor
 from .utils import _check_shuffle
 
-DEFAULT_PROJECTOR_KWARGS = {
```

Comment:
Why do we need to change tracin (and trak) for adding logra?

Reply:
Thanks for the clarification. When you get everything ready in the end, maybe it's worth opening a clean new PR to have a clearer change log.